accord-core/src/main/java/accord/local/SerializerSupport.java

```diff
@@ -88,6 +87,7 @@ public class SerializerSupport
     ImmutableSet.<MessageType>builder()
                 .addAll(PRE_ACCEPT_STABLE_TYPES)
                 .addAll(APPLY_TYPES)
+                .add(PROPAGATE_OTHER_MSG)
```

Comment:
is this true?  `accord.messages.Propagate#type`

```
    @Override
    public MessageType type()
    {
        // TODO (now): this logic doesn't work now we permit upgrading; need to pick the maximum *possible* status we might propagate
        //     might be better to
        switch (achieved.propagatesStatus())
        {
            case Applied:
            case PreApplied:
                if (toEpoch >= committedExecuteAt.epoch())
                    return MessageType.PROPAGATE_APPLY_MSG;
            case Committed:
            case Stable:
                return MessageType.PROPAGATE_STABLE_MSG;
            case PreCommitted:
                if (!achieved.definition.isKnown())
                    return MessageType.PROPAGATE_OTHER_MSG;
            case PreAccepted:
                return MessageType.PROPAGATE_PRE_ACCEPT_MSG;
            default:
                return MessageType.PROPAGATE_OTHER_MSG;
        }
    }
```
    
 if we reach applied we would be `PROPAGATE_STABLE_MSG` or `PROPAGATE_APPLY_MSG`.  This means that the only cases where this happens are below


![Screenshot 2024-06-10 at 1 06 25 PM](https://github.com/apache/cassandra-accord/assets/1852601/d53ba749-170f-4caa-8a4b-c126bcc7a454)

Reply:
We should only maybe propagate `Truncated` or `Invalidated` out of those

Reply:
Then what is the code in `SerializerSupport` supposed to be doing with `PROPAGATE_OTHER_MSG`? Because right now it's doesn't process it at all.

I can create a new type for truncated or invalidated, but then PROPAGATE_OTHER_MSG will be a no-op again.

Reply:
It's already referenced here https://github.com/apache/cassandra-accord/blob/trunk/accord-core/src/main/java/accord/local/SerializerSupport.java#L415 and seems to assume everything might be missing.

In this case what needed to propagate was invalidation.